fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -192,12 +212,13 @@ fun TabItem(
             }
         }
 
+        val indicatorCornerRadiusSize = FluentGlobalTokens.CornerRadiusTokens.CornerRadiusCircle.value
+        val indicatorWidth = FluentGlobalTokens.SizeTokens.Size160.value
         Column(
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.Center,
             modifier = modifier
                 .then(clickableModifier)
-                .background(backgroundColor)
```

Comment:
instead of removing the background and indication add a param "showHandle" and based on this flag handle the backgrounf and indication